__tests__/index.test.js

```diff
@@ -101,7 +70,24 @@ describe("nc news", () => {
               .expect(400)
               .then((response) => {
                 expect(response.body.msg).toBe('Invalid request');
-              });
-          });
+            });
+        });
+    })
+    describe("/api/articles", ()=> {
+        test("200: serves an array of all articles sorted by date in desc order", () => {
+            return request(app)
+                .get("/api/articles")
+                .expect(200)
+                .then(({body}) => {
+                  expect(body.articles.length >= 1).toBe(true)
```

Comment:
Nice, but there is an alternative way to do this which is to use the matcher `.toBeGreaterThan(...)`